backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/price/BpkPrice.kt

```diff
@@ -22,12 +22,14 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import net.skyscanner.backpack.compose.price.internal.BpkPriceAlignEnd
 import net.skyscanner.backpack.compose.price.internal.BpkPriceAlignStart
+import net.skyscanner.backpack.compose.price.internal.BpkPriceRight
 import net.skyscanner.backpack.compose.price.internal.BpkPriceRow
 
 enum class BpkPriceAlign {
     Start,
     End,
     Row,
+    Right,
```

Comment:
rather than adding a new one the `End` one needs to be updated (we use start/end on android to work with the rtl language)